src/main/java/ch/njol/skript/hooks/spark/expressions/ExprCpuUsage.java

```diff
@@ -0,0 +1,84 @@
+package ch.njol.skript.hooks.spark.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.hooks.spark.SparkHook;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import me.lucko.spark.api.Spark;
+import me.lucko.spark.api.SparkProvider;
+import me.lucko.spark.api.statistic.StatisticWindow;
+import me.lucko.spark.api.statistic.types.DoubleStatistic;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("CPU Usage")
+@RequiredPlugins("Spark")
+@Description("Returns the CPU usage readings, like the information from Spark's /tps command. " +
+			"This expression is only supported with servers that have Spark on their server.")
+@Examples({
+	"broadcast cpu usage from the last 10 seconds",
+	"broadcast cpu usage from the last 1 minute",
+	"broadcast cpu usage from the last 15 minutes",
+	"broadcast cpu usage"
+})
+@Since("INSERT VERSION")
+public class ExprCpuUsage extends SimpleExpression<Number> {
+
+	private int index;
+	private String expr = "cpu usage";
+
+	static {
+		Skript.registerExpression(ExprCpuUsage.class, Number.class, ExpressionType.SIMPLE,
+			"[the] cpu usage from [the] last 10[ ]s[econds]",
+			"[the] cpu usage from [the] last ([1] minute|1[ ]m[inute])",
+			"[the] cpu usage from [the] last 15[ ]m[inutes]",
+			"[the] cpu usage");
+	}
```

Comment:
You should use a method to check whether Spark exists to register the expression itself, not in the `init`